### DISCUSSION TITLE
Graph with many annotations is hard to read after migrating panel from Graph (old): option for adjusting annotation line width?

### DISCUSSION THREAD

**Discussion by laundmo** [grafana/grafana#44716] [Dashboards, Panels & Visualizations]
At my place of work, we recently updated Grafana from version 6 to 8.

We tend to have a lot of annotations, and panels are still using the Graph (old) visualization look just fine:
![graph with thin annotation lines](https://user-images.githubusercontent.com/24855949/151983873-82333882-f6a8-4df1-95bc-fbe82f8abee3.png)

But after migrating to the Timeseries visualization, the annotations lines get larger and obscure the graph to a way larger degree
![graph with wide annotation lines](https://user-images.githubusercontent.com/24855949/151984005-2a389d2c-a070-48da-899e-9492a551547d.png)

Therefore, I would like to ask for a (ideally global) setting to adjust their width, as to make our heavily annotated graphs more readable.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
